internal/config/server_configuration.go

```diff
@@ -20,21 +25,35 @@ func (s ControlConfiguration) Merge(config Mergable) Mergable {
 		return s
 	}
 
-	if s.Database.Url == "" {
-		s.Database.Url = sat_config.Database.Url
+	if s.Database.PostgresUrl == "" {
```

Comment:
Just a general comment but this pattern would be ideally abstracted somehow. Is Go capable of reflection on it's structs?

Reply:
yeah, but it’s a faff, and not worthwhile here (as this code’s going to be removed anyway).

if we did need to do this, using generate would probably be nicer than reflection, but that’s just speculation on my part